FAIL
Kernel panic in run: PANIC at ../../threads/interrupt.c:361 in intr_handler(): Unexpected interrupt
Call stack: 0x8004213877 0x8004208acc 0x8004208e7e 0x8004211488 0x8004216a4b 0x8004210d7a 0x800420a378 0x8004217dde 0x8004216cb3 0x8004206636 0x8004206783 0x8004206120
Translation of call stack:
0x0000008004213877: debug_panic (lib/kernel/debug.c:32)
0x0000008004208acc: intr_handler (threads/interrupt.c:365)
0x0000008004208e7e: intr_entry (threads/intr-stubs.o:?)
0x0000008004211488: __vprintf (lib/stdio.c:282)
0x0000008004216a4b: vprintf (lib/kernel/console.c:123)
0x0000008004210d7a: printf (lib/stdio.c:80)
0x000000800420a378: lock_release (threads/synch.c:280)
0x0000008004217dde: test_priority_donate_one (tests/threads/priority-donate-one.c:41)
0x0000008004216cb3: run_test (tests/threads/tests.c:57)
0x0000008004206636: run_task (threads/init.c:252)
0x0000008004206783: run_actions (threads/init.c:297)
0x0000008004206120: main (threads/init.c:125)
